src/__tests__/CountryCard.test.js

```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import CountryCard from '../components/CountryCard';
+import { store } from '../redux/store';
+
+it('renders CardComponent correctly', () => {
+  const tree = renderer
+    .create(
+      <BrowserRouter>
+        <Provider store={store}>
+          <CountryCard />
+        </Provider>
+
+      </BrowserRouter>,
+
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
🙏Please fix this test as it is failing. The received output does not🤷‍♂️ match the expected snapshot. Kindly ensure that they match to ensure the correctness✔ of your project. This will perfect your project and ensure that your CountryCard 🏚component is properly tested.